Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -199,7 +199,8 @@ void JitArm64::FallBackToInterpreter(UGeckoInstruction inst)
 
   Interpreter::Instruction instr = Interpreter::GetInterpreterOp(inst);
   MOVP2R(ARM64Reg::X8, instr);
-  MOVI2R(ARM64Reg::W0, inst.hex);
+  MOVP2R(ARM64Reg::W0, Interpreter::getInstance());
+  MOVI2R(ARM64Reg::W1, inst.hex);
```

Comment:
Why was this added/changed?

Reply:
`Interpreter::Instruction` was changed from `void (*)(UGeckoInstruction inst)` to `void (*)(Interpreter& interpreter, UGeckoInstruction inst)`, this adjusts the ARM JIT parameters to that.

Reply:
Seems like this is causing an error
MOVP2R is erroring "Can't store pointers in 32-bit registers" on Android

Reply:
Oh right, that should be X0 I guess. I don't have a 64 bit ARM device so I'm flying kinda blind on the ARM JIT changes.

Reply:
Does #11676 fix it?

Reply:
That fixes it 👍

Reply:
Great, thanks for catching that!